lib/defopt.py

```diff
@@ -991,11 +998,17 @@ def _make_literal_parser(literal, parsers, value=None):
             value, ', '.join(map(repr, map(str, _ti_get_args(literal))))))
 
 
-def _make_store_tuple_action_class(tuple_type, member_types, parsers):
-    parsers = [_get_parser(arg, parsers) for arg in member_types]
+def _make_store_tuple_action_class(tuple_type, member_types, parsers, *, is_variable_length=False):
+    if is_variable_length:
+        parsers = [_get_parser(member_types[0], parsers)]
```

Comment:
I think I would prefer
```python
if is_variable_length:
    parsers = itertools.repeat(_get_parser(member_types[0], parsers))
```
and don't modify parsers in `__call__` below.  (This way, at least in theory, one could reuse the same parser multiple times -- this is not currently possible in defopt, but one could easily imagine an API where an ArgumentParser instance is returned and the user can call `parse_args` on it multiple times.)

Reply:
Ah yeah good point